src/io/iter_csv.cc

```diff
@@ -130,29 +130,74 @@ class CSVIter: public IIterator<DataInst> {
 DMLC_REGISTER_PARAMETER(CSVIterParam);
 
 MXNET_REGISTER_IO_ITER(CSVIter)
-.describe(R"code(Iterating on CSV files
+.describe(R"code(Returns the CSV file iterator.
 
-Assume there is CSV file at ``data/data.csv`` with content::
+In this function, `data_shape` parameter is used to set the shape of each line of the input data.
+If a row in an input file is `1,2,3,4,5,6` and `data_shape` is (3,2), each row
+in csv file will be reshaped to the data [[1,2],[3,4],[5,6]] of shape (3,2).
```

Comment:
there's a jump from "a row in an input" to "each row". Stick with the example of a single row:

"If a row in an input file is `1,2,3,4,5,6` and `data_shape` is (3,2), that row
 will be reshaped, yielding the array [[1,2],[3,4],[5,6]] of shape (3,2)."